collector/bonding_linux.go

```diff
@@ -31,6 +32,7 @@ type bondingCollector struct {
 
 func init() {
 	Factories["bonding"] = NewBondingCollector
+	CollectorsEnabledState["bonding"] = flag.Bool("collectors.bonding.enabled", false, "enables bonding-collector")
```

Comment:
I did that at first as well, but then we have the issue that those flags are not compiled in based on architecture, but always.
Because the map lives in the collector-package, not in the main package, we can just add the flags into the collector-files and have the nice property that collectors that are not compiled in (e.g. mdadm_linux.go on BSDs) don't show up in `-help`. This was the least ugly way I found to accomplish this, so if you have a nice idea for that, feel free to elaborate. :)